### PR TITLE
Phone number formatting

### PR DESCRIPTION
**Pull request by laddhoffman**

### CODE REVIEW

**Comment by laddhoffman** [comment]
Closes #23

File: helpers/sheeter.js [hunk 1]
```diff
@@ -80,15 +81,23 @@ function addToSheet(type, ...data){
...
   // Make the request.
   return requestPromise(requestOptions)
     .catch(function(error){
       // Get the actual response to the post.
       // The Google app is weird and redirects the response location
       // for some reason.
-      return requestPromise({
-        uri: error.response.headers.location
-      })
+      if (error.response && error.response.statusCode === 302) {
```
Comment (pandafulmanda) [inline helpers/sheeter.js:92]: good catch!

File: routes/api/v1/twilio.js [hunk 2]
```diff
@@ -35,6 +37,14 @@ const smsText = [
   }
 ]
 
+function formatPhoneNumber(input) {
```
Comment (pandafulmanda) [inline routes/api/v1/twilio.js:40]: seems like a useful function that may come in handy in multiple files, maybe move to like a `helpers/utils.js` file later or whatever makes sense

File: routes/api/v1/twilio.js [hunk 3]
```diff
@@ -156,10 +168,11 @@ function getLanguageString(language, title, data) {
 function matchYes(req, language) {
   var input = req.body.Body.toUpperCase().trim();
   var yes = getLanguageYes(language, true);
+  debug('matchYes, input:', input);
   if (_.isArray(yes)) {
     return _.contains(yes, input);
   } else {
-    return input === getLanguageYes(language);
+    return input === yes;
```
Comment (pandafulmanda) [inline routes/api/v1/twilio.js:175]: nice, but i wonder if the configuration definitions should just have `yes` be an array regardless of whether there's only one `yes` (i.e. `yes: ["YES"]`) or multiple.  perhaps a future change

File: routes/api/v1/twilio.js [hunk 4]
```diff
@@ -47,8 +57,10 @@ const smsText = [
  * opts.value Cookie value to store
  */
 function reply(req,res,opts){
-  if (opts.key !== undefined){
-    res.cookie(opts.key, opts.value,{"path":""})
+  if (opts.data !== undefined){
```
Comment (pandafulmanda) [inline routes/api/v1/twilio.js:60]: nice 😄

**Review by pandafulmanda — APPROVED** [review]
I'm ready to merge, looks awesome!  just a couple of comments, but all stuff we can address later/as needed.

**Pull request merged**